packages/next/src/server/app-render/make-get-server-inserted-html.tsx

```diff
@@ -37,7 +40,10 @@ export function makeGetServerInsertedHTML({
           ) : null
         )
       } else if (isRedirectError(error)) {
-        const redirectUrl = getURLFromRedirectError(error)
+        const redirectUrl = addPathPrefix(
```

Comment:
should this use [`addBasePath`](https://github.com/vercel/next.js/blob/canary/packages/next/src/client/add-base-path.ts#L6)?

Reply:
I think that util is more targeting client side as it's relying on `__NEXT_MANUAL_CLIENT_BASE_PATH` which is not inlined in next-server runtime but user land's code bundle.

It would work here but not in the case in app-render. I was using the same code from how app-render handle redirect error. Ideally we could merge the handling together later if they're same.